src/Generator/Generators/ExtensionMethods.cs

```diff
@@ -23,5 +26,14 @@ public static Interop.CallingConvention ToInteropCallConv(this CallingConvention
 
             return Interop.CallingConvention.Winapi;
         }
+
+        public static bool IsParamPrimToRefTypeConvertible(Parameter param, bool checkParamUsage = true)
+        {
+            List<PrimitiveType> AllowedToHaveDefaultPtrVals = new List<PrimitiveType> {PrimitiveType.Bool, PrimitiveType.Double, PrimitiveType.Float,
```

Comment:
@tritao said he wanted to review your code too a little later. In the mean time - since you have to wait anyway - could you please change this to lower-case ('allowedToHaveDefaultPtrVals ') as per our convention for local variables.